distro/windows/Makefile

```diff
@@ -100,7 +100,7 @@ ot-config: ot-check-compiler
 	  -DCMAKE_INSTALL_PREFIX=$(OT_PREFIX) \
 	  -DPYTHON_SITE_PACKAGES=Lib/site-packages \
 	  -DUSE_TBB=OFF \
-	  -DUSE_SPHINX=OFF \
+	  -DUSE_SPHINX=OFF -DUSE_EXPRTK=OFF \
```

Comment:
I fail to see how it is related to this PR

Reply:
Oops!